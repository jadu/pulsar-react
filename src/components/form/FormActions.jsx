import React, { Children } from 'react';

export default class FormActions extends React.Component {

  static defaultProps = {
    className: 'form__actions'
  };

  render() {
    let {
      children,
      ...props
    } = this.props;

    return (
      <div {...props}>
        {children}
      </div>
    );
  }
}