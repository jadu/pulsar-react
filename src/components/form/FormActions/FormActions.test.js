'use strict';

import React from 'react';
import { render } from 'enzyme';

import FormActions from './FormActions';

it('Renders the form actions wrapper', () => {
  const tree = render(
    <FormActions>foo</FormActions>
  );
  expect(tree).toMatchSnapshot();
});
