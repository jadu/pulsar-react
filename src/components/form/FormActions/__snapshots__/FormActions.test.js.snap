// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the form actions wrapper 1`] = `
<div
  class="form__actions"
>
  foo
</div>
`;
